js/modifiedEvents.js

```diff
@@ -0,0 +1,27 @@
+export function eventIsModified(e) {
+  return e.which > 1 ||
+    e.shiftKey ||
+    e.altKey ||
+    e.metaKey ||
+    e.ctrlKey;
+}
+
+/**
+ * Allow shift-click, ctrl-click, etc. in browsers to work as intended when they
+ * might otherwise be swallowed by event handlers
+ */
+export function handleModifiedClicks() {
+  document.addEventListener('click', e => {
+    // Ignore events that aren't modified link clicks
+    if (e.target.nodeName != 'A' || !eventIsModified(e)) return;
```

Comment:
This breaks when clicking `<a><span>link</span></a>`. You'll have to use something like `e.target.closest?.('a')``